packages/iTwinUI-react/src/core/Select/SelectTag.tsx

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import cx from 'classnames';
+import React from 'react';
+import SvgCloseSmall from '@itwin/itwinui-icons-react/cjs/icons/CloseSmall';
+import { useTheme, CommonProps } from '../utils';
+import '@itwin/itwinui-css/css/inputs.css';
+
+export type SelectTagProps = {
+  /**
+   * Callback function that handles click on close icon.
+   * Close icon is shown only when this function is passed.
+   * Use only with 'default' Tag.
+   */
+  onRemove?: React.MouseEventHandler;
+  /**
+   * Text inside the tag.
+   */
+  children: React.ReactNode;
+} & CommonProps;
+
+/**
+ * Tag for showing selected value in `Select`.
+ */
+export const SelectTag = (props: SelectTagProps) => {
```

Comment:
Not sure about this name `SelectTag`. Have you looked at other libs?

Reply:
Also it doesn't look very customizable to me, so does the user even need it? Or if it's private, that's fine by me.

Reply:
Name up for discussions (would be nice to change it in css too then).
I was not planning to expose this to users.. So havent thought about customizations (unless it might be needed?)
Now also got idea, should we start marking these components with additoinal tags like `@private` or `@internal`. 
We dont do it everywhere, I think

Reply:
I see antd calling them just `tag` and mui says its `chip`.
react-select I found only `multi-value`
atlsassian design is using react-select for demo, so `multi-value` too

so idk.. I like tag or chip? just we cannot use simply `tag`, so I udnerstand `SelectTag` is confusing :/

Reply:
If we mark it as private, I'm fine with whatever name, including `SelectTag`. Just maybe improve the description a little to mention multi select